style.css

```diff
@@ -582,15 +583,22 @@ label {
     background-image: url(assets/background/Header-llustration-desktop.svg);
   }
 
+  .main p {
+    width: 85%;
+    max-width: 768px;
+    margin-top: 1%;
+  }
+
   .span__br {
     display: none;
   }
 
-  p,
-  .main p {
-    width: 85%;
-    max-width: 768px;
-    margin-top: 10%;
+  .projects_h2_icon {
+    display: none;
+  }
+
+  .social_media ul {
+    margin-bottom: 5%;
```

Comment:
- [ ] I would kindly suggest you to must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).the section has three background image both left and rignt and on your design only one background image is display please fix this,as the background not display as the original design figma teplate  kindly fix this background image to make your website completly great and professional .
- [ ] your design
![Screenshot 2023-06-10 031644](https://github.com/mateoburvi/Portfolio/assets/92458236/de382c05-4e21-4f2f-9cf6-0ff39dbf6b92)

- [ ] figma design
![Screenshot 2023-06-10 030236](https://github.com/mateoburvi/Portfolio/assets/92458236/9d4830c6-c486-44d7-b70d-207a0431bb40)